### DISCUSSION TITLE
alist-windows-amd64-upx.zip 和 alist-windows-amd64.zip有啥区别？

### DISCUSSION THREAD

**Discussion by rrq0514** [AlistGo/alist#5122] [Q&A] [answered]
### Please make sure of the following things

- [ ] I have read the [documentation](https://alist.nn.ci).
- [ ] I'm sure there are no duplicate issues or discussions.
- [ ] I'm sure this feature is not implemented.
- [ ] I'm sure it's a reasonable and popular requirement.

### Description of the feature / 需求描述

如题

**Comment by mengguyi** [comment, marked as answer]
The difference between `alist-windows-amd64-upx.zip` and `alist-windows-amd64.zip` is that the former is compressed using the UPX executable packer which reduces the size of the executable file. The latter is not compressed using UPX. Both files are available for download on the [official GitHub repository of Alist](https://github.com/alist-org/alist)
